packer_files/nutanix_centos_template/centos-config/ks.cfg

```
@@ -0,0 +1,58 @@
+# Use graphical install
+text
+
+%packages
+@^server-product-environment
+kexec-tools
+openssh-server
+perl
+cloud-init
+%end
+
+services --enabled=sshd
+
+url --url="http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/"
+repo --name="AppStream" --baseurl="http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/"
+
+# Keyboard layouts
+keyboard --xlayouts='us'
+# System language
+lang en_US.UTF-8
+
+# Setting up network interface to DHCP
+network --bootproto=dhcp --ipv6=auto --hostname=centos-ks.local --activate
+
+# Run the Setup Agent on first boot
+firstboot --enable
+
+ignoredisk --only-use=sda
+# System bootloader configuration
+bootloader --append="crashkernel=auto" --location=mbr --boot-drive=sda
+
+part /boot --fstype=ext4 --size=512
+part pv.1 --size=10000 --grow
+volgroup vg00 pv.1
+logvol swap --vgname=vg00 --recommended --name=swap --fstype=swap
+logvol / --vgname=vg00 --size=5000 --grow --name=root --fstype=ext4
+
+# Partition clearing information
+clearpart --none --initlabel
+
+# System timezone
+timezone America/New_York --isUtc
+
+# Root password
+sshkey --username=root "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC8L9EzpTcR0VZTf2Pqt/qrdtIeMZoqm2a+KvVHG7XdCDj4DW3d/8bLPduIm6GmvhU0F09H1VYfTvAviB5CMM7FVmfVEH112wZhUVjpcuPf75/1on9xernbYiysqKmc8pPaadlko7NQfFc36LrV6UpUpZoFUqd7FdZLbi6hLT41tQqRiNto+TbyfxtusyB/eynrxLaRVPA5+Jmah4jir9L0j7m2e9bn3om+rXBPY1zlp5pds+xYwfkRWaXJ93G2s1QlpicNBJOckAVgEOZddHDGjGqtFCt7v58nEF53dgpdDk26qKeqiavRdQ5NDRXMNPL/YlqVz/EG4HBtL628BvKCVuok910olIqEr8sQK5WTla1WMmiEjByYX2w6tSoAgvmfwczshko0Qq8h4oX/rmn4TDjpgnXSQvaCaq4IePcANrd7wQSgSQmXnglP2HuotLhE3+/FVXdu39/ZJDAHk9G3ACa2JxgMZTe6UlfI9uAP4yCOpvhspvLadq5GTzDysabFpuWli5F5BEpLjalt+HHBKnQzFtjthY8qZ+Ey71Z1XK/f7H+kQ7D22H08AQu7ccmana7bkT6wsauRdfojyveMmapI8DL8iit7+liX/n6+58wSrpjsozHetaQ0dQjAYUdqYDFs4YZBMomoL4xFaBpLUuVqmVwrzbgRNVX7miu1uw== vrutkovs@redhat.com"
```

maybe we should do the same as we are doing on vsphere:

```suggestion
sshkey --username=root "${key}"
```

And set `key` on `sources.pkr.hcl`

---

Nutanix terraform won't let me create a file on CD-ROM from (templated) contents. I think I can fix it by creating a local file, but  that's something I could fix in the follow up PR